helm/prometheus-rules/templates/platform/atlas/alerting-rules/grafana-cloud.rules.yml

```
@@ -29,6 +29,23 @@ spec:
         severity: page
         team: atlas
         topic: observability
+  - name: resource-usage
+    rules:
+    - alert: IncorrectResourceUsageData
+      annotations:
+        description: '{{`Data used in the Grafana Cloud Resource Usage dashboard is incorrect for cluster {{ $labels.cluster_id }}.`}}'
```

Actually the opsrecipe explains a lot!

```suggestion
                # More info about this alert can be found in the opsrecipe: https://intranet.giantswarm.io/docs/support-and-ops/ops-recipes/resource-usage-dashboard/
                description: '{{`Data used in the Grafana Cloud Resource Usage dashboard is incorrect for cluster {{ $labels.cluster_id }}.`}}'
```

---

Isn't this obvious as there's already the ops-recipe link below ?

---

Didn't seem obvious to me that the opsrecipe would not only explain how to fix the issue, but also how the alert is written, and why we consider it's an important issue.

Maybe that should be an opsrecipe standard, but AFAIK very few of ours do that :sweat_smile:

---

I think adding a comment on top of the alert that explain what is it and how it works makes sense for all alerts though, because going to the ops-recipe all the time is not the most straigthforward access to the information

---

Added the comment